mimic/model/clb_objects.py

```diff
@@ -97,7 +101,7 @@ def as_json(self):
         """
         :return: a JSON dictionary representing the node.
         """
-        return attr.asdict(self)
+        return dissoc(attr.asdict(self), "feed_events")
```

Comment:
This is so easy to write in pure Python that I'm hesitant to want to include the library.

```python
attrs = attr.asdict(self)
return dict([(key, attrs[key]) for key in attrs if key != "feed_events"])
```